stdlib/src/builtin/int.mojo

```diff
@@ -18,7 +18,7 @@ These are Mojo built-ins, so you don't need to import them.
 from collections import KeyElement
 
 from builtin.hash import _hash_simd
-from builtin.string import _calc_initial_buffer_size
+from builtin.string import _calc_initial_buffer_size, atol
```

Comment:
**Suggestion** From the `builtin` module, you only need to explicitly export _private_ entities (those prefixed with an `_`).  In this case, `atol` is public, so there's no need to explicitly import it.  It (and any public entity in the `builtin` module) will always be imported into every Mojo program.